service/package.json

```diff
@@ -21,6 +21,8 @@
   },
   "homepage": "https://github.com/z1to/MavenlinkReportBuilder#readme",
   "devDependencies": {
+    "@types/axios": "^0.14.0",
+    "@types/dotenv": "^8.2.0",
```

Comment:
I get these warnings because of these 2 packages:
<img width="1280" alt="Screen Shot 2021-04-10 at 3 58 10 PM" src="https://user-images.githubusercontent.com/10281272/114283684-f11c7a00-9a18-11eb-97b9-ef8fa57dac00.png">

I tried uninstalling them and everything works perfectly without them. No warnings either.